dlt/helpers/ibis.py

```diff
@@ -109,11 +109,14 @@ def _ignore_hstore(conn: Any, name: Any) -> Any:
         con = ibis.snowflake.connect(
             schema=dataset_name, **sn_credentials, create_object_udfs=False
         )
-    elif issubclass(destination.spec, MsSqlClientConfiguration):
+    elif issubclass(destination.spec, MsSqlClientConfiguration) and not issubclass(
```

Comment:
note: not sure whats going on here, pls verify yourself (just to say I wont invest time to understand this now. hard to say why I would expect this to be a correct change without more context.)